package.json

```diff
@@ -12,6 +12,7 @@
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint",
+    "prepare": "husky install",
```

Comment:
husky already runs `husky install` when it's installed via `npm i`

Reply:
It did do that, but they stopped doing it on the last major release. For a while, they recommended using the npm `post-install` action to do it automatically, but even that has now been discouraged as it is considered an anti-pattern by npm.

Reply:
alright sounds good then! 👍

Reply:
I might have spoken too soon there... I was under the impression `prepare` script was not running automatically, but I just removed my "node_modules" folders and when I did `npm i` to get it back, I got the same message that on your screenshot!

Reply:
I moved from `prepare` to `bootstrap` to ensure it is an opt-in only feature!